docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -1,129 +1,92 @@
+    ```
 
-4.  Create a folder for the Device Service that we are going to develop.
-    In this step, you are naming the folder to the name you want to give
-    your new Device Service. Standard practice in EdgeX is to prefix the
-    name of a Device Service with `device-`:
+4.  Copy the example code from **device-sdk-go** to **device-simple**:
+    ``` bash
+    cd ~/go/src/github.com/edgexfoundry
+    cp -rf ./device-sdk-go/example/* ./device-simple/
+    ```
 
-        mkdir device-simple
+5.  Copy Makefile to device-simple:
+    ``` bash
+    cp ./device-sdk-go/Makefile ./device-simple
+    ```
+6. Copy version.go to device-simple:
+    ``` bash
+    cp ./device-sdk-go/version.go ./device-simple/
+    ```
 
-5.  Copy the example code from **device-sdk-go** to **device-simple**:
+After completing these steps, your device-simple folder should look like the listing below.
 
-        cp -rf ./device-sdk-go/example/* ./device-simple/
+![image](EdgeX_GettingStartedSDKCopyFiles.png)
 
-    ![image](EdgeX_GettingStartedSDKClone5.png)
+## Start a new Device Service
 
-6.  Copy Makefile to device-simple:
+With the device service application structure in place, time now to program the service to act like a sensor data fetching service.
 
-        cp ./device-sdk-go/Makefile ./device-simple
+1.  Change folders to the device-simple directory.
 
-    ![image](EdgeX_GettingStartedSDKClone6.png)
+    ``` bash
+    cd ~/go/src/github.com/edgexfoundry/device-simple
+    ```
 
-7.  Copy VERSION to device-simple:
+2.  Open main.go file in the cmd/device-simple folder with your favorite text editor. Modify the
+    import statements.  Replace `github.com/edgexfoundry/device-sdk-go/example/driver` with `github.com/edgexfoundry/device-simple/driver` in the import statements. Also replace `github.com/edgexfoundry/device-sdk-go` with `github.com/edgexfoundry/device-simple`. Save the file when you have finished editing.
 
-        cp ./device-sdk-go/VERSION ./device-simple/
+    ![image](EdgeX_GettingStartedSDKReplaceImports.png)
 
-    ![image](EdgeX_GettingStartedSDKClone7.png)
-
-8.  Copy version.go to device-simple:
-
-        cp ./device-sdk-go/version.go ./device-simple/
-
-## Starting a new Device Service project
-
-The device-sdk-go comes with example code to create a new Device
-Service. Complete the following steps to modify the copy of the example
-code to use in your new service.
-
-1.  Edit the main.go file in the cmd/device-simple folder. Modify the
-    import statements to replace "device-sdk-go/example/driver" to
-    "device-simple/driver" from the paths in the import statements. Save
-    the file when you have finished editing.
-
-    ![image](EdgeX_GettingStartedSDKProject6.png)
-
-2.  Open Makefile in your favorite text editor and make the following
+3.  Open Makefile found in the base folder (~/go/src/github.com/edgexfoundry/device-simple) in your favorite text editor and make the following
```

Comment:
This seems really overkill to me -- and irrelevant for creating a device service. Why should we enforce the communities' build strategies on others? I think we keep it simple with `go build` and `go run`

Reply:
Same answer here.  Makefile is there right now.  Let's fix after we just have it working.